Tasks/IISWebAppDeploymentOnMachineGroup/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "vsts-tasks-iiswebappdeploymentonmachinegroup",
+  "version": "1.0.0",
+  "description": "VSTS  IIS WebApp Deployment On Machine Group",
+  "main": "DeployIISWebApp.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/Microsoft/vsts-tasks.git"
+  },
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Microsoft/vsts-tasks/issues"
+  },
+  "homepage": "https://github.com/Microsoft/vsts-tasks#readme",
+  "dependencies": {
+    "winreg": "^1.2.2",
+    "vsts-task-lib": "^0.9.7",
+    "q": "^1.4.1",
+    "adal-node": "^0.1.15",
+    "xml2js": "^0.4.13",
+    "adm-zip": "^0.4.7",
+    "gulp": "^3.9.1",
+    "gulp-zip": "^3.2.0",
```

Comment:
remove modules that are not directly consumed in main task files. And do this for azure rm task as well